hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/datanode/DecommissionStatusSubCommand.java

```diff
@@ -83,6 +85,8 @@ public void execute(ScmClient scmClient) throws IOException {
       DatanodeDetails datanode = DatanodeDetails.getFromProtoBuf(
           node.getNodeID());
       printDetails(datanode);
+      Map<String, List<ContainerID>> containers = scmClient.getContainersOnDecomNode(datanode);
```

Comment:
Maybe should add a simple test, or extend an existing test in `TestDecommissionStatusSubCommand`. I am not sure if it will fail or not at the moment as it may not have a mock defined for the new `scmClient.getContainersOnDecomNode(datanode)` call.

Reply:
I have extended the tests in TestDecommissionStatusSubCommand to check for this as well now